### DISCUSSION TITLE
Send a clickable Link to add Event to your calendar?

### DISCUSSION THREAD

**Discussion by Apfelkomplott** [liedekef/events-made-easy#278] [Q&A] [answered]
Hey Franky.

I was asked to provide a clickable link after registering to an event, where you cann easily add the Event (Date & Location at least) to your own used calendar.
Of course I searched the forum, but didn't find a hint.
Hope, there is already a way?

Thanks in advance

Alex.

**Comment by liedekef** [comment, marked as answer]
Use #_ICALLINK : displays a link for an ICAL representation of the event
